README.md

```diff
@@ -21,16 +21,16 @@ Puppet module to configure wireguard through systemd-networkd configs
 ## Setup
 
 The module can create firewall rules with [voxpupuli/ferm](https://github.com/voxpupuli/puppet-ferm#puppet-ferm).
-This is enabled by default can be disabled by setting the `manage_firewall`
+This is enabled by default but can be disabled by setting the `manage_firewall`
 parameter to false in the `wireguard::interface` defined resource. You need to
-have the `ferm` class in your catalog to use the feature.
+have the `ferm` class in your catalogue to use the feature.
```

Comment:
I'm fine with the changes except for catalog->catalogue, because catalog is the official term in the Puppet docs.